docs/overview.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: Overview
 > Call for translators! [We're looking for translators](https://github.com/all-contributors/all-contributors/issues/143) to help translate this spec for everyone!
 
 <div align="center">
-    <img src="../assets/logo-full-transparent.png" alt="✨ All Contributors ✨" width="800px" />
+    <img src="../docs/assets/logo-full-transparent.png" alt="✨ All Contributors ✨" width="800px" />
```

Comment:
Have you checked if the deployed version still shows the image like on https://allcontributors.org/docs/en/overview?

Reply:
The deployed version showed the image, but it was not visible in the markdown file.

Reply:
I would have suggested seeing what the Netlify preview shows but it's blocked due to the semantic PR check.